indico/modules/designer/util.py

```diff
@@ -69,3 +83,11 @@ def get_default_badge_on_category(category, only_inherited=False):
     parent_chain = category.parent_chain_query.options(joinedload('default_badge_template')).all()
     return next((category.default_badge_template for
                  category in reversed(parent_chain) if category.default_badge_template), None)
+
+
+def get_badge_format(tpl):
+    format_map_landscape = {name: (h, w) for name, (w, h) in FORMAT_MAP_PORTRAIT.items()}
+    format_map = format_map_landscape if tpl.data['width'] > tpl.data['height'] else FORMAT_MAP_PORTRAIT
```

Comment:
How about this instead (or use ternary if it doesn't look too bad with the rapping)? 

```python
if tpl.data['width'] > tpl.data['height']:
    format_map = {name: (h, w) for name, (w, h) in FORMAT_MAP_PORTRAIT.items()}
else:
    format_map = FORMAT_MAP_PORTRAIT
```

Reply:
Done this too 👍